src/bindings/cpp/onion.hpp

```diff
@@ -26,6 +26,7 @@
 
 #include <string>
 #include <onion/onion.h>
+#include "listen_point.hpp"
```

Comment:
As it is using LitenPoint& there is no need to fully include listen_point. It is enough with a forward class declaration "class ListenPoint;" (inside the namespace). This speeds up compilation times.